teps/0092-scheduling-timeout.md

```diff
@@ -371,7 +373,7 @@ https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-conditions
 Cons:
 * The situations a user might want to address here (i.e. image pull backoff, long image pulls, contrained resource
   environment) seem like they're all fairly distinct, and the values for timeouts for each of these (e.g. a few seconds
-  only for image pull backoff, maybe 5 minutes for scheduling in a resource constrained environment) users would want to 
+  only for image pull backoff, maybe 5 minutes for scheduling in a resource constrained environment) users would want to
   provide might be very different, so bundling them into one option doesn't seem like the best way to address them
 
 Introducing the scheduling timeout initially and only including the time before the pod has the condition `PodScheduled`
```

Comment:
Since we've changed the proposal a bit, I'm not sure that it still leaves open options to add more timeouts later. The original idea was that the scheduling timeout would run before all other timeouts, but now we're introducing a set of timeouts that must all add up to the total. So it's not clear that we could still add a meaningful "imagepullbackoff" timeout later.

Reply:
I think the section here about being able to add a new timeout later on can just be deleted.